rideshare_nguyen.rb

```diff
@@ -0,0 +1,153 @@
+########################################################
+# Diana Nguyen - Space
+# Ride Share
+########################################################
+# (0) Setting up the entire data structure:
+# Level 1: master hash "rides"
+# Level 2: 4 sub hashes for each of the drivers
+#         driver id is the key and the values are different
+#         variables (e.g. date of ride they gave)
+# Level 3: within the values hold the array for the
+#         variables, this is so the variables line up,
+#         (e.g. for Driver 1, they had a date of ride of
+#          Feb 3rd, with a rating of 3, earning $10, and 
+#          the rider had an id of RD0003)
+
+rides = {
+  :DR0001 => {
+  date_of_ride: ["3rd Feb 2016", "3rd Feb 2016", "5th Feb 2016"],
+  rating: [3, 4, 2],
+  money_earned: [10, 30, 45],
+  rider_of_drive: ["RD0003", "RD0015", "RD0003"]
```

Comment:
If you are going to align these keys and values like this, indent these lines. It makes it more clear what is and isn't in each hash in `rides`